src/lexedata/edit/add_central_concepts.py

```diff
@@ -1,5 +1,6 @@
 import typing as t
 import collections
+import logging
```

Comment:
I have so far used `cli.logging`. I don't mind, but we should be consistent.

Reply:
Then let's better go with `cli.logging`